CHANGES.rst

```diff
@@ -31,6 +31,9 @@ Changes to be released in next version
  * MXSession: [MXSession decryptEvent:inTimeline:] is deprecated, use [MXSession decryptEvents:inTimeline:onComplete:] instead.
  * MXCrypto: [MXCrypto decryptEvent:inTimeline:] is deprecated, use [MXCrypto decryptEvents:inTimeline:onComplete:] instead.
  * MXCrypto: [MXCrypto hasKeysToDecryptEvent:] is now asynchronous.
+ * MXRoomSummary: lastMessageEventId, lastMessageOriginServerTs, isLastMessageEncrypted, lastMessageString, lastMessageAttributedString, lastMessageOthers properties removed (vector-im/element-ios/issues/4360).
```

Comment:
We can remove this line now. Those properties have just been moved and it is explained in the line.
We should have a line saying that `MXRoomSummary.lastMessageEvent` is gone for performance reasons.

Reply:
👍